policy/modules/contrib/daemontools.if

```diff
@@ -43,7 +43,7 @@ interface(`daemontools_service_domain',`
 		type svc_run_t;
 	')
 
-	domain_auto_trans(svc_run_t, $2, $1)
+	domain_auto_transition_pattern(svc_run_t, $2, $1)
```

Comment:
For all of these: have you considered using `domtrans_pattern()` instead? Seems more appropriate.

Reply:
Hm... the extra permissions seem logical, but I don't know the history of these interfaces and refpolicy also replaced it with `domain_auto_transition_pattern()`, so I just went with the safe (equivalent) choice for this PR.